litmus/VULKAN/DR/fencefence2-filter.litmus

```diff
@@ -0,0 +1,13 @@
+Vulkan fencefence2
+{
+P1: r0 = 0;
+P0: r1 = 0;
+x=0;
+y=0;
+}
+ P0@sg 0, wg 0, qf 0                 | P1@sg 1,wg 0, qf 0                  ;
+ st.av.dv.sc0 x, 1                   | ld.atom.wg.sc1 r0, y                ;
+ membar.rel.wg.semsc01               | membar.acq.wg.semsc01               ;
```

Comment:
Are you sure that the `membar`s should have `semsc01` and not both `semsc0` and `semsc1`  as [here](https://github.com/KhronosGroup/Vulkan-MemoryModel/blob/master/alloy/tests/fencefence2.test#L9)

Reply:
Using `semsc0` and `semsc1` make more sense. The `semsc01`, althought mentioned in the alloy model, seems not be used in all tests and the [parser](https://github.com/KhronosGroup/Vulkan-MemoryModel/blob/master/alloy/litmus.cpp#L54-L55).

Reply:
Interesting. Why don't you open an issue in their repo asking them what is the purpose of this set if it is never used in their parser?

Reply:
Just found it seems the same as `ACQ_REL` case, based on their parser [here](https://github.com/KhronosGroup/Vulkan-MemoryModel/blob/master/alloy/spirv.als#L270).

Reply:
Then you need to add this definition to the memory model, allow our parser to have more than one `semsc` and rewrite the unit tests with things like `membar.rel.wg.semsc0.semsc1` similar to their tests

Reply:
Updated semsc to be a list